.github/workflows/function-tests-template.yml

```diff
@@ -0,0 +1,162 @@
+name: OpenSearch Dashboards Function Tests Template
+
+on:
+  workflow_call:
+    inputs:
+      endpoint:
+        required: true
+        type: string
+    secrets:
+      osd-user:
+        required: true
+      osd-user-password:
+        required: true
+
+env:
+  # TODO: merge all the changes to OpenSearch-Project's function repo.
```

Comment:
could we create a github issue for track?

Reply:
Yes. opened a issue and added here.